### ISSUE TITLE
create-react-app build - Failed to minify src/utils/normalizeKey.js

### ISSUE THREAD

**Issue by loopmode** [localForage/localForage#808]
In an ejected create-react-app, I want to use localforage.

In the development server, everything worked right away - I did not need to use `noParse`.

When I try to `yarn build` however, something strange happens:

```
$ node scripts/build.js                                              
Creating an optimized production build...                            
Failed to compile.                                                   
                                                                     
Failed to minify the code from this file:                            
                                                                     
        ./node_modules/localforage/src/utils/normalizeKey.js:4       
                                                                     
Read more here: http://bit.ly/2tRViJ9                                
                                                                     
error An unexpected error occurred: "Command failed.                 
```

The funny thing is that it's a `/src/` file! I have no idea how why that is imported by webpack there. The "main" field in localforage's package.json clearly points to `dist/localforage.js`.

So, that's basically the issue. Sorry for not providing more details, however there's a simple workaround:

```
import localforage from 'localforage/dist/localforage';
```

Then webpack has no doubts and it works.

**Comment by tofumatt** [comment]
Yes, this is because of #805. Because we are an ES6-based project we recently added the field `mainFields` that webpack will use to include the file.

Unfortunately it seems our use of template strings in that file causes Webpack builds to fail; our config on addons-frontend had to be changed not to use this import.

It appears it may be causing issues. I wonder if it's better to revert that changes and allow people to include the src file directly if they choose in their own webpack config. 🤷‍♂️

**Comment by tofumatt** [comment]
The newly-released 1.7.1 should fix this, sorry about that!

**Comment by loopmode** [comment]
1.7.1 does fix it. Thanks!
